ompi/mca/coll/han/coll_han_component.c

```diff
@@ -160,22 +170,37 @@ static bool is_simple_implemented(COLLTYPE_T coll)
     }
 }
 
+/**
+ * topo level conversions both ways; str <-> id
+ * An enum is used for conversions.
+ */
+static mca_base_var_enum_value_t level_enumerator[] = {
+    { INTRA_NODE, "intra_node" },
+    { INTER_NODE, "inter_node" },
+    { GLOBAL_COMMUNICATOR, "global_communicator" },
+    { 0 }
+};
+
 /*
  * Stringifier for topological level
  */
-const char* mca_coll_han_topo_lvl_to_str(TOPO_LVL_T topo_lvl)
+const char* mca_coll_han_topo_lvl_to_str(TOPO_LVL_T topo_lvl_id)
```

Comment:
I realize t hat this breaks the common naming scheme. It should be `ompi_coll_han_topo_lvl_to_str`. Given that this has been merged into main already this (and other occurrences) should be fixed in a separate PR.